config/api.yml

```diff
@@ -724,6 +724,39 @@
       - :name: delete
       :delete:
       - :name: delete
+  :custom_button_sets:
+    :description: Custom Button Sets
+    :identifier: ab_buttons_accord
+    :options:
+    - :collection
+    :verbs: *gpppd
```

Comment:
Can you also add a test for the third p - PATCH? 😁

Reply:
Added in b6ee82d